### PR TITLE
Unit test implementation #2

### PR DESCRIPTION
**Pull request by joaorighetto**
I'm creating this new pull request to include the changes from implementing the new custom admin command

### CODE REVIEW

File: __project__/fixtures/sample.yaml [hunk 1]
```diff
@@ -3,43 +3,6 @@
 # This file defines initial data to be used on application setup
 # See https://docs.djangoproject.com/en/4.1/howto/initial-data/.
```
Comment (artu-hnrq) [inline __project__/fixtures/sample.yaml:5]: Why would us stop dealing with these fixtures?

File: event/management/commands/generate_event_data.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+class Command(BaseCommand):
+    help = (
+        "Create events with specified quantities of past and upcoming events."
+        f"If quantity is not provided creates {DEFAULT_QUANTITY} events."
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument("quantity", type=int, nargs="?")
+        parser.add_argument("-u", "--upcoming", type=int, default=0)
+        parser.add_argument("-p", "--past", type=int, default=0)
```
Comment (artu-hnrq) [inline event/management/commands/generate_event_data.py:25]: Give a `help` description to each command's argument is very welcome!

File: event/managers.py [hunk 3]
```diff
@@ -11,9 +11,16 @@ def get_queryset(self):
                 is_active=ExpressionWrapper(
```
Comment (artu-hnrq) [inline event/managers.py:11]: I'd say we wouldn't need it anymore.

File: event/tests/factories.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import factory
+
+from factory import Faker
+
+from faker_optional import OptionalProvider
+
+from event.models import Event
+
+
+Faker.add_provider(OptionalProvider)
```
Comment (artu-hnrq) [inline event/tests/factories.py:12]: Consider have these configuration on a centralized `compat` module

File: event/tests/test_models.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    """
+
+    validity = data.pop("validity")
+    serializer = EventSerializer(data=data)
+
+    return serializer.is_valid() is validity
+
+
+@pytest.mark.django_db
+def test_event_active_manager_method(event_factory):
```
Comment (artu-hnrq) [inline event/tests/test_models.py:33]: Change to deal with upcomming events

File: event/tests/test_models_cases.py [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+def random_past_date():
+    return fake.past_datetime(tzinfo=dt.UTC)
+
+
+def random_future_date():
+    return fake.future_datetime(tzinfo=dt.UTC)
+
+
+def create_event_data_from_key_values(blank_key=None, null_key=None):
```
Comment (artu-hnrq) [inline event/tests/test_models_cases.py:25]: Consider use `factory_boy` as data generation engine

**Review by artu-hnrq — CHANGES_REQUESTED** [review]
We are on the way!

File: __project__/fixtures/sample.yaml (hunk 1 above)
Comment (joaorighetto) [reply __project__/fixtures/sample.yaml:5]: They were interfering with the tests

**Review by artu-hnrq — APPROVED** [review]
Baitão

**Pull request merged**